test/core/util/test_config.cc

```diff
@@ -135,26 +97,51 @@ static void print_stack_from_context(CONTEXT c) {
 #endif
 
   HANDLE process = GetCurrentProcess();
-  HANDLE thread = GetCurrentThread();
 
   SYMBOL_INFOW* symbol =
       (SYMBOL_INFOW*)calloc(sizeof(SYMBOL_INFOW) + 256 * sizeof(wchar_t), 1);
   symbol->MaxNameLen = 255;
   symbol->SizeOfStruct = sizeof(SYMBOL_INFOW);
 
-  while (StackWalk(imageType, process, thread, &s, &c, 0,
-                   SymFunctionTableAccess, SymGetModuleBase, 0)) {
-    BOOL has_symbol =
-        SymFromAddrW(process, (DWORD64)(s.AddrPC.Offset), 0, symbol);
-    fwprintf(
-        stderr, L"*** %016I64X %ls - %016I64X\n", (DWORD64)(s.AddrPC.Offset),
-        has_symbol ? symbol->Name : L"<<no symbol>>", (DWORD64)symbol->Address);
+  const unsigned short MAX_CALLERS_SHOWN =
+      8192;  // avoid flooding the stderr if stacktrace is way too long
+  for (int frame = 0; frame < MAX_CALLERS_SHOWN &&
+                      StackWalk(imageType, process, thread, &s, &c, 0,
+                                SymFunctionTableAccess, SymGetModuleBase, 0);
+       frame++) {
+    PWSTR symbol_name = L"<<no symbol>>";
```

Comment:
There's an outstanding portability build failure that looks relevant

"""
T:\src\github\grpc\workspace_c++_windows_dbg_native_default_default_buildonly\test\core\util\test_config.cc(126): error C2440: '=': cannot convert from 'PCHAR' to 'PWSTR' [T:\src\github\grpc\workspace_c++_windows_dbg_native_default_default_buildonly\cmake\build\grpc_test_util.vcxproj]
"""